index.bs

```diff
@@ -169,11 +207,71 @@ This allows developers to detect support for paint timing for a particular [=bro
 Processing model {#sec-processing-model}
 ========================================
 
+Recording paint timing {#sec-recording-paint-timing}
+--------------------------------------------------------
+
+<h4 id="sec-modifications-dom">Modifications to the DOM specification</h4>
+
+Each {{Element}} has an <dfn>associated image request</dfn> which is initially null.
+When the processing model for an {{Element}} <em>element</em> of type {{HTMLImageElement}}, {{SVGImageElement}}, or {{HTMLVideoElement}} creates a new image resource (e.g., to be displayed as an image or poster image), <em>element</em>'s <a>associated image request</a> is set to the <a>image request</a> of the created image resource.
+
+Note: Every image resource that is obtained from a URL whose <a spec=url>scheme</a> is equal to "data" has an associated <a>image request</a> which is not fetched but still needs to be loaded.
+This request can be the <a>associated image request</a> of an {{Element}}.
+
+Note: The current language is vague since it does not point to specific algorithms.
+This can be made more rigorous when the corresponding processing models have a more unified processing model.
+
+When an {{Element}} <em>element</em>'s <a>associated image request</a> has become <a>completely available</a>, run the algorithm to <a>process an image that finished loading</a> passing in <em>element</em> and its <a>associated image request</a> as inputs.
+
+<h4 id="sec-modifications-CSS">Modifications to the CSS specification</h4>
+
+When the user agent is executing the <a>painting order</a>, it must populate the <a>set of owned text nodes</a> of the painted {{Element|Elements}} so that the following is true:
```

Comment:
This is a bit strange. How is the painting order (which is about stacking context) related here? Sounds like something that belongs in an implementation

Reply:
Good question! (And yes, it was moved over from element timing, where it was added pretty early, and maybe could have had more review at the time).

"Executing the painting order" isn't really something that happens: things are painted, and the painting order defines the order that they are painted in, but nothing in that section appears to be an algorithm that gets run when painting actually happens.